Source/Core/Editor/UI/GraphView/Windows/SearchableGroupListPopup.cs

```diff
@@ -221,7 +221,7 @@ private void FilterListByPartialMatch(string searchText)
 
             foreach (VisualElement child in scrollView.Children())
             {
-                if (child.Q<Label>("Label") is Label label)
+                if (child.Q<Label>("Name") is { } label)
```

Comment:
Why do you use { } instead of Label?

Reply:
its an other null checking pattern﻿ because we don't have to check if the type is right (we do this already at child.Q<Label>, so we just make sure that the type is not null and declare a label if its not null).